kickthemout.py

```diff
@@ -737,6 +739,17 @@ def targetList(option, opt, value, parser):
         callback=targetList, type='string',
         dest='targets', help='specify target IP address(es) and perform attack')
 
+
+    def target_files(option, opt, value, parser):
+        with open(value) as file:
+            addr_list = file.readlines()
```

Comment:
- `addressFile` instead of `addr_list` (camel case)

Reply:
I update all names to camel case